weather_dl/download_pipeline/manifest.py

```diff
@@ -16,15 +16,18 @@
 import abc
 import collections
 import dataclasses
+import datetime
+from enum import Enum
```

Comment:
Optional nit: let's use `import enum` and `class Foo(enum.Enum)`.

Reply:
Done.